integration/signer/signer_int.sh

```diff
@@ -93,12 +90,27 @@ delete_occ() {
               "https://containeranalysis.googleapis.com/v1/${OCC_NAME}"
       fi
     fi
-    exit $ARG
+}
+
+deploy_image() {
+  IMAGE_URL=$1
+  POD_NAME=$2
+  gcloud config set compute/zone us-central1-c
+  gcloud container clusters get-credentials signer-int-test
+  kubectl run --generator=run-pod/v1 --image=$IMAGE_URL $POD_NAME --command ls
+}
+
+delete_pod() {
+  set +ex
```

Comment:
Shouldn't this be in the beginning of the script, not in a function like this?

Reply:
We want the testing to stop whenever a test fails and errors, but the clean-up to always run to completion even some cleaning step fails.